FiveCalls/FiveCalls/Colors.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Colors.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 8/25/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+
+extension Color {
+    static let FiveCallsLightBG = Color(red: 0.85, green: 0.85, blue: 0.85)
+    static let FiveCallsDarkBlue = Color(R.color.darkBlue()!)
```

Comment:
Feel free to cherry pick my change [here](https://github.com/5calls/ios/pull/337/commits/6a1262ed8b0d7d55358c76b689e80f78081d797e) to allow removing this extension all together.

Reply:
Turns out Xcode 15 has a new setting that will generate color extensions for you! I prefixed the color asset names so it's clear which ones are nonstandard. I tried the same thing with the new xcstrings support, but I'm not sure I like how automatic it is... I prefer seeing the autocompletion for existing strings like R.swift shows.

Reply:
Nice! I see you updated some colors still using R, would we be able to convert all the colors off of R, or is there still a use case?

Reply:
I don't plan on making improvements to the UIKit code where we can avoid it, we can get rid of it during the dependency cleanup phase. SwiftUI stuff should all be the new Xcode color extensions though, I'll take a look if I missed anything.